deel/lip/activations.py

```diff
@@ -87,23 +87,19 @@ def __init__(
     ):
         """
         GroupSort activation
-
```

Comment:
Are you sure that you want to remove these docstring line breaks? In other docstrings, there are present.

Reply:
probably removed by black/pycharm reformat. As it passed linting I didn't notice it :-)